resolwe_bio/processes/support_processors/insert_size.py

```diff
@@ -0,0 +1,117 @@
+"""Collect metrics about the insert size of a paired-end library."""
+import os
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    FloatField,
+    Process,
+    SchedulingClass,
+    StringField
+)
+
+
+class InsertSizeMetrics(Process):
+    """Collect metrics about the insert size of a paired-end library.
+
+    Tool from Picard, wrapped by GATK4. See GATK
+    CollectInsertSizeMetrics for more information.
+    """
+
+    slug = 'insert-size'
+    name = 'InsertSizeMetrics'
+    process_type = 'data:picard:insert'
+    version = '1.0.0'
+    shaduling_class = SchedulingClass.BATCH
+    entity = {'type': 'sample'}
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/dnaseq:4.2.0'
+            }
+        },
+    }
+    data_name = '{{ bam|sample_name|default("?") }}'
+
+    class Input:
+        """Input fields for InsertSizeMetrics."""
+
+        bam = DataField('alignment:bam', label='Alignment BAM file')
+        genome = DataField('genome:fasta', label='Genome')
+        minimum_fraction = FloatField(
+            label='Minimum fraction of reads in a category to be considered',
```

Comment:
Multiline strings may need an extra space at the end (i.e. see line 66) to properly format spaces between words.